core/src/main/scala/spinal/core/Spinal.scala

```diff
@@ -167,7 +167,7 @@ case class SpinalConfig(mode                           : SpinalMode = null,
                         removePruned                   : Boolean = false,
                         allowOutOfRangeLiterals        : Boolean = false,
                         dontCareGenAsZero              : Boolean = false,
-                        obfuscateNames                 : Boolean = false,
+                        obfuscateNamesPrefix           : String  = null,
```

Comment:
how about set this to "" empty string to avoid errors?

Reply:
Since I'm a newbie of scala, can you explain which kind of errors will it leads to ?  D:)

Reply:
I think we should keep as it was before.
obfuscateNames is the feature
obfuscateNamesPrefix is a configuration for the feature, there can be more to comes.

So i think we should keep obfuscateNames, and provide a default value for obfuscateNamesPrefix ("oo_")

Reply:
I totally agree.